src/robot/parsing/model/statements.py

```diff
@@ -214,6 +242,16 @@ def alias(self):
 class ResourceImport(Statement):
     type = Token.RESOURCE
 
+    @classmethod
+    def from_params(cls, resource, separator=FOUR_SPACES, eol=EOL):
+        tokens = [
+            Token(Token.RESOURCE, 'Resource'),
+            Token(Token.SEPARATOR, separator),
+            Token(Token.NAME, resource),
+            Token(Token.EOL, eol)
+        ]
+        return cls(tokens)
```

Comment:
Yeah I also noticed that I'm switched the way I call cls() -> I will go with cls([]) whenever I define token list directly, without any further changes so it will be consistent.